src/js/components/FormField/FormField.js

```diff
@@ -226,7 +226,7 @@ if (process.env.NODE_ENV !== 'production') {
   FormFieldDoc = require('./doc').doc(FormField); // eslint-disable-line global-require
 }
 const FormFieldWrapper = compose(
-  withFocus,
+  withFocus({ focusWithMouse: true }),
```

Comment:
I didn't notice any changes in the UI behavior for FormField (e.g [FormField-TextInput](https://storybook.grommet.io/?selectedKind=FormField&selectedStory=TextInput&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel)) 
was it intended to be addressed?

Reply:
pushed a fix for this